### PR TITLE
Threads rename v2

### PR DESCRIPTION
**Pull request by Maxtors**

### CODE REVIEW

File: src/util-device.c [hunk 1]
```diff
@@ -106,6 +109,28 @@ char *LiveGetDeviceName(int number)
...
+    size_t devnamelen = strlen(devname);
+
+    if (devnamelen > MAX_DEVNAME) {
+        strncpy(newdevname, devname, DEVNAME_CHUNCK);
+        strncpy(newdevname+DEVNAME_CHUNCK, "...", 3);
+        strncpy(newdevname+8, devname+(devnamelen-DEVNAME_CHUNCK), DEVNAME_CHUNCK);
+        strncpy(newdevname+13, "\0", 1);
+        SCLogInfo("Shortening device name to: %s", newdevname);
+    } else {
+        strcpy(newdevname, devname);
```
Comment (inliniac) [inline src/util-device.c]: banned function

File: src/util-device.c [hunk 2]
```diff
@@ -106,6 +109,28 @@ char *LiveGetDeviceName(int number)
...
+ *  \param device name from config and destination for modified
+ *
+ *  \retval None, is added to destination char *newdevname
+ */
+void LiveSafeDeviceName(const char *devname, char *newdevname)
+{
+    size_t devnamelen = strlen(devname);
+
+    if (devnamelen > MAX_DEVNAME) {
+        strncpy(newdevname, devname, DEVNAME_CHUNCK);
```
Comment (inliniac) [inline src/util-device.c]: strncpy is banned as well, please use strlcpy

File: src/util-device.c [hunk 3]
```diff
@@ -106,6 +109,28 @@ char *LiveGetDeviceName(int number)
...
+        strncpy(newdevname+DEVNAME_CHUNCK, "...", 3);
```
Comment (inliniac) [inline src/util-device.c]: strlcat is meant for adding strings after another

File: src/util-device.c [hunk 4]
```diff
@@ -106,6 +109,28 @@ char *LiveGetDeviceName(int number)
 }
 
 /**
+ *  \brief Shorten a device name that is to long
+ *
+ *  \param device name from config and destination for modified
+ *
+ *  \retval None, is added to destination char *newdevname
+ */
+void LiveSafeDeviceName(const char *devname, char *newdevname)
```
Comment (inliniac) [inline src/util-device.c]: I think we should pass a destination length parameter here

**Comment by Maxtors** [comment]
Added new PR: 1421

**Pull request closed without merging**